README.md

```diff
@@ -129,6 +129,111 @@ It is _critical_ that you require the user to be logged in and fetch the
 user identifier and email (e.g. `$user->getid()` and `$user->getEmail()`)
 from that authenticated user (not from anywhere in the URL).
```

Comment:
we need to probably just remove this paragraph.

Reply:
done